env/bulid.go

```diff
@@ -1,5 +1,7 @@
 package env
```

Comment:
Not related but the file should be `build.go`, not `bulid.go`

Reply:
thx